packs/st2ci/actions/mistral.yaml

```diff
@@ -8,11 +8,9 @@ parameters:
   test_host:
     type: string
     description: The name of the hosts where the integration test(s) will run.
-    default: None
```

Comment:
Actually I have clearly read this wrong but so will confirm once I can get mistral running on the build box.